ownCloud/Client/SortBar.swift

```diff
@@ -97,7 +97,7 @@ class SortBar: UIView, Themeable {
 	func applyThemeCollection(theme: Theme, collection: ThemeCollection, event: ThemeEvent) {
 		self.sortButton.applyThemeCollection(collection)
 		self.sortSegmentedControl.applyThemeCollection(collection)
-		self.applyThemeCollection(collection)
+		self.backgroundColor = Theme.shared.activeCollection.navigationBarColors.backgroundColor
```

Comment:
Please use `collection` instead of `Theme.shared.activeCollection` here.

Reply:
I'll change it.